travis-ci.sh

```diff
@@ -63,6 +66,33 @@ if [[ $PARTS =~ (^|,)tests(,|$) ]]; then
     done
 fi
 
+# MongoDB tests starting dummy server which can be analyzed exactly
+if [[ $PARTS =~ (^|,)mongo(,|$) ]]; then
+    mongod --version
+
+    for ex in $(\ls -1 tests/mongodb); do
+        if [ -r tests/mongodb/$ex/run.sh ]; then
+            echo "[INFO] Running mongo test $ex"
+            (cd tests/mongodb/$ex && DUB_INVOKE="dub --compiler=$DC --override-config=vibe-d:core/$VIBED_DRIVER $DUB_ARGS" ./run.sh)
+        elif [ -r tests/mongodb/$ex/dub.json ] || [ -r tests/mongodb/$ex/dub.sdl ]; then
+            MONGOPORT=22824
+            rm -f tests/mongodb/log.txt
+            rm -rf tests/mongodb/$ex/db
+            mkdir -p tests/mongodb/$ex/db
+            MONGOPID=$(mongod --logpath tests/mongodb/log.txt --bind_ip 127.0.0.1 --port $MONGOPORT --dbpath tests/mongodb/$ex/db --fork | grep -Po 'forked process: \K\d+')
+
+            echo "[INFO] Running mongo test $ex"
+            (cd tests/mongodb/$ex && dub --compiler=$DC --override-config=vibe-d:core/$VIBED_DRIVER $DUB_ARGS -- $MONGOPORT && dub clean && mongodump --port=$MONGOPORT)
+
+            if [ -r tests/mongodb/$ex/test.sh ]; then
+                (cd tests/mongodb/$ex && ./tests/mongodb/$ex/test.sh)
+            fi
+
+            kill $MONGOPID
```

Comment:
Why do we have to duplicate this into `travis-ci.sh`?
This will make running/reproducing the tests a lot harder locally.

Reply:
hm I always run travis-ci.sh locally to test vibe.d and having the PARTS there makes it easy to make it only run the mongodb tests

Reply:
added comments why